composer.json

```diff
@@ -20,21 +20,22 @@
     "sort-packages": true
   },
   "require": {
-    "php": "^7.4 | ^8.0",
+    "php": "^8.1",
     "ext-curl": "*",
     "ext-json": "*",
-    "symfony/cache": "^4.4 | ^5.2",
-    "symfony/config": "^4.4 | ^5.2",
-    "symfony/dependency-injection": "^4.4 | ^5.2",
-    "symfony/http-client": "^4.4 | ^5.2",
-    "symfony/http-foundation": "^4.4 | ^5.2",
-    "symfony/http-kernel": "^4.4 | ^5.2",
-    "symfony/security-guard": "^4.4 | <5.3"
+    "symfony/cache": "^6.0",
+    "symfony/config": "^6.0",
+    "symfony/dependency-injection": "^6.0",
+    "symfony/http-client": "^6.0",
+    "symfony/http-foundation": "^6.0",
+    "symfony/http-kernel": "^6.0",
+    "symfony/security-core": "^6.0",
+    "symfony/security-http": "^6.0"
```

Comment:
Why not `^6.1` packages?

Reply:
Because there are no features in `6.1` that we depend on. It's not the responsibility of the library to push for never versions